### DISCUSSION TITLE
how to test?

### DISCUSSION THREAD

**Discussion by sawirricardo** [saloonphp/saloon#60] [Q&A]
Hi, thanks for this package. I want to ask, I have a laravel package here https://github.com/sawirricardo/laravel-icd-api, and want to test instantiation an object. however, when instantiating the object, I need to make a request call, to icd.who.int. How to mock this? Thank you

**Comment by sawirricardo** [comment]
Basically, I think I'm looking at like Http::fake(), where we can pass any links concerned. How to do that?

**Comment by Sammyjo20** [comment]
Hey! So I see you are using Laravel/Orchestra test bench so this should be relatively easy.

Before you instantiate your SDK, you can provide `Saloon::fake()` which will prepare Saloon with fake responses. There are various ways you can fake, I would recommend the "Request" fake. Take a look here and let me know if you get stuck or it's not quite the solution

https://docs.saloon.dev/next-steps/testing-faking-responses/laravel

**Comment by Sammyjo20** [comment]
Hey there, did this help?
